app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,18 @@
+class ProjectsController < ApplicationController
+  def show
+    @project = Project.find(params[:id])
+  end
+
+  def add
+    @project = Project.find(params[:id])
+    @contestant = Contestant.find(contestant_id)
+    ContestantProject.create(contestant: @contestant, project: @project)
+    redirect_to "/projects/#{@project.id}"
+  end
+
+  private
+  def contestant_id
+    require "pry"; binding.pry
+    # CANT SEEM TO SEE WHERE THE FILLED_IN DATA IS TO FIND THE CONTESTANT WITH THE MATCHING ID
+  end
```

Comment:
It looks like you have your text_field_tag as `:id` and not `:contestant_id`.. this may have caused an issue when creating a ContestantProject. Nice job realizing that you needed to add to the joins table. To make this more restful, consider putting everything that you have in that `add` action into the ContestantProjectsController, in a `create` action.